onnxruntime/core/framework/sequential_executor.cc

```diff
@@ -456,9 +456,9 @@ onnxruntime::Status ExecuteThePlan(const SessionState& session_state, const std:
                                       std::vector<OrtValue>& fetches,
                                       const std::unordered_map<size_t, IExecutor::CustomAllocator>& fetch_allocators,
                                       const logging::Logger& logger,
-                                      const DeviceStreamColloection& device_streams,
+                                      const DeviceStreamCollection& device_streams,
                                       const bool& terminate_flag,
-                                      const bool only_execute_path_to_fetches,
+                                      const bool /*only_execute_path_to_fetches*/,
```

Comment:
i added the support of only_execute_path_to_fetches in latest PE branch, so you probably don't need to comment it out now.